pyproject.toml

```diff
@@ -18,21 +18,26 @@ terra-scientific-pipelines-service-api-client = "0.1.29"
 python-dotenv = "^1.0.1"
 click = "^8.1.7"
 colorlog = "^6.9.0"
-oauth2-cli-auth = "^1.5.0"
+oauth2-cli-auth = "1.5.0"
 PyJWT = "^2.9.0"
 requests = "^2.32.3"
 tqdm = "^4.67.0"
 tzlocal = "^5.2"
 tabulate = "^0.9.0"
 poetry = "1.8.5"
+cryptography = "^44.0.1"
```

Comment:
to address https://github.com/DataBiosphere/terra-scientific-pipelines-service-cli/security/dependabot/3 (not related to this PR)